config/locales/en.yml

```
@@ -520,6 +520,18 @@ en:
       show_count: Applications (%{count})
     statistics: Statistics
 
+  candidate_profiles_interstitials:
+    page_title: You can now view candidate profiles and invite them to apply to jobs
+    intro: A candidate's profile includes their job preferences, experience and qualifications.
+    list_header: >-
+      These candidates have specficied:
```

We probably don't need this to be a block of text, and there's a typo here.
```suggestion
    list_header: These candidates have specified:
```